app/server/lib/DocApi.ts

```diff
@@ -1169,6 +1172,19 @@ export class DocWorkerApi {
       }
     }));
 
+    // POST /api/docs/:docId/disable
+    // Disables doc (similar to soft-delete, but needs admin access)
+    this._app.post('/api/docs/:docId/disable', requireInstallAdmin, async (req, res) => {
```

Comment:
Why are `/enable` and `/disable` implemented in different locations? Aren't they similar?